packages/loader/container-loader/src/container.ts

```diff
@@ -96,6 +98,8 @@ import { parseUrl, convertProtocolAndAppSummaryToSnapshotTree } from "./utils";
 
 const detachedContainerRefSeqNumber = 0;
 
+const connectEventName = "connect";
```

Comment:
ideally we should be unified in the codebase on using "connected" (which is used everywhere besides container and delta manager looks like) instead of "connect" here :/

Reply:
"connect" != "connected" - these are two different events (used in this layer).
"connect" is not exposed outside of loader.
Current shape is not ideal, but that's the design we have right now